index.css

```diff
@@ -388,3 +403,213 @@ label {
   font-weight: 400;
   line-height: 28px;
 }
+
+.list_style {
+  list-style: none;
+}
+
+@media only screen and (min-width: 768px) {
+  .nav1 {
+    display: none;
+  }
+
+  .nav2 {
+    display: block;
+    display: flex;
+    gap: 20px;
+  }
+
+  .nav2 a {
+    font-size: 15px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .section1 {
+    display: flex;
+    justify-content: center;
+  }
+
+  .section1 h1 {
+    font-size: 48px;
+    color: #172b4d;
+    line-height: 60px;
+    font-weight: 700;
+    width: 420px;
+  }
+
+  .section1 p {
+    color: #344563;
+    font-size: 13px;
+    line-height: 28px;
+    width: 580px;
+    font-weight: 400;
+    align-items: left;
+    padding: 50px 0 0 0;
+  }
+
+  .works_section {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 90%;
+    margin: auto;
+    padding: 20px 0 30px 0;
+  }
+
+  .works_section .article1,
+  .works_section .article2,
+  .works_section .article3,
+  .works_section .article4 {
+    display: flex;
+    justify-content: center;
+    gap: 15px;
+    margin: 20px 0;
+  }
```

Comment:
Please adjust the width of your work_section articles 😀:
| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/199867506-17c40726-e98b-4e7c-a798-3836af2a5b6a.png)  | ![image](https://user-images.githubusercontent.com/42799579/199867519-7773aa7a-2c87-4fdb-8e62-20687b9bafbf.png)  |

Notice that in the original the card occupies almost 80% of the screen and the spacing of the elements inside it is larger 😉.